pkg/s3wrapper/client.go

```diff
@@ -341,3 +346,70 @@ func (c *S3Client) handleObject(ctx context.Context, log logrus.FieldLogger, obj
 		}
 	}
 }
+
+func (c *S3Client) ListObjectsByPrefix(ctx context.Context, prefix string) ([]string, error) {
+	log := logutil.FromContext(ctx, c.log)
+	var objects []string
+	log.Info("Listing objects by with prefix %s", prefix)
+	resp, err := c.client.ListObjects(&s3.ListObjectsInput{
+		Bucket: aws.String(c.cfg.S3Bucket),
+		Prefix: aws.String(prefix),
+	})
+	if err != nil {
+		err = errors.Wrapf(err, "Error listing objects for prefix %s", prefix)
+		log.Error(err)
+		return nil, err
+	}
+	for _, key := range resp.Contents {
+		objects = append(objects, *key.Key)
+	}
+	return objects, nil
+}
+
+func (c *S3Client) DownloadListOfFiles(ctx context.Context, files []string) (io.ReadCloser, int64, error) {
```

Comment:
Please name the returned params so it's easier to understand what is being returned.
What the int64 param is?

Reply:
content lenght